win32/devdoc/job_object_helper.md

```diff
@@ -30,6 +33,65 @@ MOCKABLE_FUNCTION(, THANDLE(JOB_OBJECT_HELPER), job_object_helper_create);
 **SRS_JOB_OBJECT_HELPER_18_019: [** `job_object_helper_create` shall succeed and return the `JOB_OBJECT_HELPER` object.  **]**
 
 
+## job_object_helper_create_with_name
+```c
+MOCKABLE_FUNCTION(, THANDLE(JOB_OBJECT_HELPER), job_object_helper_create_with_name, const char*, job_name);
+```
+``job_object_helper_create_with_name`` is used to create a Job Object with the specified name.

```

Comment:
>`` [](http://example.com/codeflow?start=0&length=2)

Probably only one backtick needed here

Reply:
Likely copied from line 19 which has double backticks as well (and should be fixed while we're here)